docs/UserGuide.md

```diff
@@ -988,18 +1108,19 @@ You may want to **re-enter your client data in a fresh JSON file** in the event
+| `e/Email`                                                             | <ul><li>Emails should be of the format local-part@domain.</li><li>The local-part should only contain alphanumeric characters and these special characters: +_.- (excluding parentheses).</li><li>The local-part may not start or end with any special characters.</li><li>This is followed by a '@' and then a domain name, made up of domain labels separated by periods.</li><li>The domain name must end with a domain label at least 2 characters long.</li><li>Each domain label must start and end with alphanumeric characters.</li><li>Each domain label may consist of alphanumeric characters, separated only by hyphens, if any.</li><li>Top-level domain (TLD) such as .com are not compulsory, as according to Internet Protocol Standards (<a href="https://datatracker.ietf.org/doc/html/rfc5322#section-3.4.1">RFC 5322 Section 3.4.1</a>).</li></ul> | <ul><li>✅ <code>e/admin@realodex</code></li><li>❌ <code>e/hello@x_mail</code></li></ul>                               |
+| `a/Address`                                                           | <ul><li>Must not include other command prefixes (`a/`,`b/`,`e/`,`f/`,`h/`,`i/`,`n/`,`p/`,`r/`,`t/`) to prevent parsing errors. For instance, `a/lemontree street t/1` may cause the command to fail, as the system will interpret `t/` as an unintended tag prefix.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | <ul><li>✅ <code>a/6 College Avenue West</code></li>                                                                   |
```

Comment:
```suggestion
| `a/Address`                                                           | <ul><li>Must not include other command prefixes with spaces before (`a/`,`b/`,`e/`,`f/`,`h/`,`i/`,`n/`,`p/`,`r/`,`t/`) to prevent parsing errors. For instance, `a/lemontree street t/1` may cause the command to fail, as the system will interpret `t/` as an unintended tag prefix.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | <ul><li>✅ <code>a/6 College Avenue West</code></li>                                                                   |
```

Reply:
`with spaces before ` basically

Reply:
🇹🇼